src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import Img from 'gatsby-image';
+
+interface Props {
+  data: GatsbyTypes.RecipeQueryQuery;
```

Comment:
This doesn't seem to pass my TS checking locally - I think we should create a graphql fragment in the `RecipeCard.tsx` file and use that fragment name in the recipe page query.